app/controllers/support/recruitment_cycles_controller.rb

```diff
@@ -2,6 +2,8 @@
 
 module Support
   class RecruitmentCyclesController < ApplicationController
-    def index; end
+    def index
+      @recruitment_cycles = RecruitmentCycle.ordered_by_current_first
```

Comment:
Do we need a scope if is only used here? Or is the idea to use somewhere? Wdyt? @Nitemaeric

Reply:
Yes, initially me too, but adds more indirection to the code. But it sounds okish for now

Reply:
Why do we need to order by current first? Can't we `order(year: :desc)`?